pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -209,9 +211,26 @@ func (r *Builder) PodEnvironment(vmRef ref.Ref, sourceSecret *core.Secret) (env
 			Value: fingerprint,
 		},
 	)
+	if macsToIps != "" {
+		env = append(env, core.EnvVar{
+			Name:  "V2V_staticips",
+			Value: macsToIps,
+		})
+	}
 	return
 }
 
+func (r *Builder) mapMacStaticIps(vm *model.VM) string {
+	configurations := []string{}
+	for _, guestNetwork := range vm.GuestNetworks {
+		if guestNetwork.Origin != string(types.NetIpConfigInfoIpAddressOriginManual) {
```

Comment:
how about being more positive :)
```suggestion
		if guestNetwork.Origin == string(types.NetIpConfigInfoIpAddressOriginManual) {
```
and then call `append`?